http-clients/feign-common/src/main/java/org/triplea/http/client/HttpClient.java

```diff
@@ -30,7 +30,7 @@
  * @param <ClientTypeT> The feign client interface, should be an interface type that has feign
  *     annotations on it.
  */
-@Slf4j
+@Log
```

Comment:
Why java.log over slf4j?

Reply:
Fixed.
Good question, I did not notice that IntelliJ was replacing this.